lib/private/AppConfig.php

```
@@ -462,11 +464,24 @@ private function getTypedValue(
 			throw new AppConfigTypeConflictException('conflict with value type from database');
 		}
 
-		if ($lazy) {
-			return $this->lazyCache[$app][$key] ?? $default;
+		if ($lazy && isset($this->lazyCache[$app][$key])) {
+			$value = $this->lazyCache[$app][$key];
+		} elseif (!$lazy && isset($this->fastCache[$app][$key])) {
+			$value = $this->fastCache[$app][$key];
+		} else {
+			return $default;
 		}
 
-		return $this->fastCache[$app][$key] ?? $default;
+		$sensitive = $this->isTyped(self::VALUE_SENSITIVE, $knownType);
+		if ($sensitive) {
+			// Only decrypt values that are stored encrypted
+			$sections = substr_count($value, '|');
+			if ($sections === 2 || $sections === 3) {
+				$value = $this->crypto->decrypt($value);
```

what would happen if unencrypted sensitive string fullfil this condition ?
should we add a prefix to the stored value to help identify encrypted value ?
do we manage/ignore exception on decrypt ?